client/pages/pool-info.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+class PoolInfo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isLoading: true,
+      poolId: this.props.poolId,
+      info: null
+    };
+  }
+
+  componentDidMount() {
+    fetch(`api/pool/${this.state.poolId}`)
+      .then(res => res.json())
+      .then(body => {
+        this.setState({
+          info: body[0],
```

Comment:
Instead of having your serve send you a singleton array, just have it send you the thing you wanted.